scheduler/src/cook/compute_cluster.clj

```diff
@@ -50,6 +50,12 @@
   (restore-offers [this pool-name offers]
     "Called when offers are not processed to ensure they're still available.")
 
+  (get-outstanding-synthetic-pods [this pool-name]
```

Comment:
The only thing we do with this is count the result. So why not make this instead be `get-outstanding-synthetic-pod-count` and a comment that this exists only for metrics?

Reply:
We don't just count the results, it is also used on line 554.

Reply:
Can you double-check the line number?

Reply:
I did, it is used in this chunk of code:
```
                (let [using-pools? (config/default-pool)
                      synthetic-task-pool-name (when using-pools? pool-name)
                      new-jobs (remove (fn [{:keys [job/uuid]}]
                                         (some #(= (str uuid) (synthetic-pod->job-uuid %))
                                               outstanding-synthetic-pods))
```

Reply:
Here's the code on default: https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/kubernetes/compute_cluster.clj#L555 (I can't seem to link to a line within the changed MR version)